frontend/libs/studio-components/src/hooks/useTableSorting.test.tsx

```diff
@@ -0,0 +1,84 @@
+import { useTableSorting } from './useTableSorting';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+import type { Rows } from '../components';
+
+type TestComponentProps = {
+  rows: Rows;
+};
+
+type Row = Record<string, React.ReactNode> & Record<'id', string | number>;
+
+const TestComponent = ({ rows }: TestComponentProps) => {
+  const { sortedRows, handleSorting } = useTableSorting(rows);
+  return (
+    <div>
+      <button onClick={() => handleSorting('name')}>Sort by Name</button>
+      <button onClick={() => handleSorting('creator')}>Sort by Creator</button>
+      <ul>
+        {sortedRows.map((row: Row) => (
+          <li key={row.id}>
+            {row.name} - {row.creator}
+          </li>
+        ))}
+      </ul>
+    </div>
+  );
+};
```

Comment:
We can avoid creating test components by using renderHook for testing hooks. Please read docs here: https://testing-library.com/docs/react-testing-library/api/#renderhook

Reply:
Fixed 😊